ddocs/Makefile

```diff
@@ -0,0 +1,18 @@
+
+export PATH := ./node_modules/.bin:$(PATH)
+
+COUCH_URL ?= $$LOCAL_COUCH
+
+all: medic-api.json
+
+medic-api.json: medic-api/*.js
+	5984 compile medic-api/index.js > medic-api.json
+
+install: medic-api.json
+	# bug: does not fail on empty response
+	cat medic-api.json | \
+	  5984 fetch-revs "$(COUCH_URL)/medic" | \
+	  5984 bulk "$(COUCH_URL)/medic" | \
+	  tee /dev/stderr | \
+	  jq -e '.ok == true' > /dev/null
+
```

Comment:
No, just, no. Don't use makefiles when we already use grunt. That's just confusing.